Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM golang:1.6
 
-RUN go get -u github.com/segment-sources/postgres
 
-# Additionally add a cron-like runner to run on an
-# interval.
-RUN go get -u github.com/segmentio/go-every
+ADD . /go/src/github.com/segment-sources/postgres
```

Comment:
location is fine. didn't know `ADD` will `mkdir` for you... just docker build . -> docker run and it worked though!